src/test/java/org/gridsuite/cgmes/assembling/job/ProfilesAcquisitionJobTest.java

```diff
@@ -408,4 +427,21 @@ public void testAcceptedFilesConnection() throws IOException {
             e.printStackTrace();
         }
     }
+
+    public void truncate() {
+        JdbcConnector connector = cgmesAssemblingLogger.getConnector();
+        try (Connection connection = connector.getConnection();
```

Comment:
don't put the connection in the try with resource like in the rest of the code ?